Shared/ViewModels/SettingsViewModel.swift

```diff
@@ -8,16 +8,27 @@
 
 import Defaults
 import Foundation
+import JellyfinAPI
+import Stinsen
 import SwiftUI
 
-final class SettingsViewModel: ObservableObject {
+final class SettingsViewModel: ViewModel {
+
+	@RouterObject
+	var router: SettingsCoordinator.Router?
 
 	var bitrates: [Bitrates] = []
 	var langs: [TrackLanguage] = []
 
 	let server: SwiftfinStore.State.Server
 	let user: SwiftfinStore.State.User
 
+	@Published
+	var quickConnectCode = ""
+
+	@Published
+	var validQuickConnect = true
```

Comment:
To indicate errors, we instead present an alert instead of this handling error state this way. `ViewModel` already has the error variable `errorMessage` to be modified and presented to the user using `.alert`. You can see a simple example in `UserSignInView` how an error is presented. 

You will also need to indicate loading, like done in your last PR about user sign in. You will need to attach `.trackActivity(loading)` to the API call, as done in `login` in `UserSignInViewModel` for reference. I forgot to request this change to your Public Users PR to do this for the `isLoadingUsers` as well instead of handling the boolean manually. Since I mention this comment here, I'm fine if you do this now, or later, or I'll probably get to it when I do another large clean up PR. You can then reference `loading` like you've done before and disable the "Authorize" button while loading.

Reply:
I also just had the thought to keep the button disabled until a full 6 character code.

Reply:
I added the `trackActivity` for the Public Users as well